sim/examples/ping/comp.go

```diff
@@ -18,6 +18,17 @@ func (p *PingMsg) Meta() *sim.MsgMeta {
 	return &p.MsgMeta
 }
 
+func (p *PingMsg) Clone() sim.Msg {
+	return p
```

Comment:
Can we have a `PingReq` and a `PingRsp` structs. Both of them should have meta. The request should have `SeqID`. Remove `PingMsg`.